lib/src/equals_matcher.dart

```diff
@@ -70,7 +70,9 @@ class _StringEqualsMatcher extends FeatureMatcher<String> {
       _writeLeading(buff, escapedItem, start);
       _writeTrailing(buff, escapedItem, start);
       buff.write('\n          ');
-      for (var i = start > 10 ? 14 : start; i > 0; i--) buff.write(' ');
+      for (var i = start > 10 ? 14 : start; i > 0; i--) {
+        buff.write(' ');
```

Comment:
```dart
buff.write(' ' * (start > 10 ? 14 : start));
```

Reply:
Sure, but there is no test that covers this behavior, so I don't want to churn it.